tests/integration/test_vercel_logs.py

```diff
@@ -0,0 +1,352 @@
+from unittest import mock
+import json
+
+from sentry_relay.consts import DataCategory
+
+TEST_CONFIG = {
+    "outcomes": {
+        "emit_outcomes": True,
+        "batch_size": 1,
+        "batch_interval": 1,
+        "aggregator": {
+            "bucket_interval": 1,
+            "flush_interval": 1,
+        },
+    },
+    "aggregator": {
+        "bucket_interval": 1,
+        "initial_delay": 0,
+    },
+}
```

Comment:
```suggestion
TEST_CONFIG = {
    "outcomes": {
        "emit_outcomes": True,
    },
}
```

None of these are necessary. `emit_outcomes` is if you want to emit http outcomes, but also optional if you instead assert client reports.

Almost certain you can omit `TEST_CONFIG` completely in these tests.

Reply:
Without the `TEST_CONFIG` like this the outcomes were not aggregating properly in tests. Not sure if there is another part of this test I'm doing wrong.

If this is problematic, can I come back to this after this PR gets merged in? I can also try to refactor the otlp and ourlog tests.

Reply:
Are you sure, because you're just repeating the default config: https://github.com/getsentry/relay/blob/ca7e20d0a7e27d2029cfb536858d25f04065c4f3/tests/integration/fixtures/relay.py#L147-L158

This didn't used to be the default, I changed that 2 weeks ago, maybe you tried before that?

Reply:
Ah it's because of `"flush_interval": 1` vs `"flush_interval": 0`. Means I just need to change to `outcomes_consumer.get_aggregated_outcomes(n=4)`. That's my mistake!

Changed with 84b22248446239d4e4cb99b2b62940f051621024